File: Data Structures & Algorithms/concatenation-of-array/submission-1.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        var size = nums.Length;
        var ret = new int[size * 2];

        for (int i = 0; i < size; i++)
        {
            ret[i] = ret[i + size] = nums[i];
        }

        return ret;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        nxt = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[nxt - 1]:
                nums[i], nums[nxt] = nums[nxt], nums[i]
                nxt += 1
        
        return nxt